hyperspace/core/src/command.rs

```diff
@@ -111,7 +119,48 @@ impl Cmd {
 			any_chain_b.name(),
 			client_id_a_on_b
 		);
-		Ok(())
+
+		let parachain_client_a = ParachainClient::try_from(any_chain_a).unwrap();
```

Comment:
We should make this more generic, at this point we only need to update the client id in the config, so why not add a set_client_id method on `AnyConfig`>
So here we just do something like `any_config_a.set_client_id(client_id_a_on_b)`, then we can just jump to 
```let new_config = Config {
			chain_a: any_config_a,
			chain_b: any_config_b,
			core: config.core,
		};

Reply:
> set_client_id

Is it that way? I thought it was
```
		config.chain_a.set_client_id(client_id_b_on_a);
		config.chain_b.set_client_id(client_id_a_on_b);
```

Reply:
It's the opposite